src/snowcli/cli/connection.py

```diff
@@ -39,7 +40,8 @@ def _mask_password(connection_params: dict):
 
 @app.command(name="list")
 @with_output
-def list_connections() -> OutputData:
+@global_options
+def list_connections(**kwargs) -> OutputData:
```

Comment:
Should we stick to convention of using `**options` instead of `**kwargs`?

Reply:
Do we have a convention? 😱

Reply:
Changed and in few more places too